carbonmark-api/src/routes/projects/get.utils.ts

```diff
@@ -148,10 +167,14 @@ export const isValidMarketplaceProject = (
   return !!validProjects.length;
 };
 
+/** The specific CarbonOffset type from the find findDigitalCarbon query*/
+export type FindQueryDigitalCarbon =
+  FindDigitalCarbonProjectsQuery["carbonProjects"][number];
```

Comment:
If you have the fragments from earlier comment and re-generate types then this type shouldn't need to be manually created.

Reply:
couldn't figure out to use the generated types in for this use, so kept this manual version to reduce verbosity

Reply:
Ok no worries 👍